labs/b1.md

```diff
@@ -6,6 +6,8 @@ layout: lab
 Welcome to the first lab! Here are some commands you might find helpful:
 `cat, cd, emacs, file, grep, less, ls, man, nano, tar, wget, vim`
 
+It may be convenient to submit your answers to Gradescope as you go! 
+
 Remember: Google and `man` are your friends!
 
```

Comment:
Might want to add a little more instruction here about labs in general, given that it's the first lab. (e.g. labs are graded for completion not correctness, labs are due x days/weeks after the assign date, if you have any questions ask in the slack channel, etc.)